packages/playground/wordpress/src/version-detect.ts

```diff
@@ -1,13 +1,20 @@
-import type { PHPRequestHandler } from '@php-wasm/universal';
+import type { PHP, PHPRequestHandler } from '@php-wasm/universal';
 import { SupportedWordPressVersions } from '@wp-playground/wordpress-builds';
 
 export async function getLoadedWordPressVersion(
 	requestHandler: PHPRequestHandler
 ): Promise<string> {
 	const php = await requestHandler.getPrimaryPhp();
+	return getWordPressVersionFromPhp(php);
```

Comment:
It's a little strange to be going from requestHandler to php and back to requestHandler when calling `getWordPressVersionFromPhp()` here.

Reply:
I agree. It's reverted now [efb4b57](https://github.com/WordPress/wordpress-playground/pull/1532/commits/efb4b57787c8e6cbcbe3792b9c5f6ca206404125)